design-documents/graph-ql/coverage/search.md

```diff
@@ -0,0 +1,79 @@
+# Queries
+
+```graphql
+type Query {
+    multiSearch(phrase: String!, productSize: Int = 10): MultiSearchResponse!
```

Comment:
I don't know that `multiSearch` and `productSearch` have the clearest names - it's hard to tell without digging through their types what their purposes are.

Can we rename them or include a description of their intended uses?

Reply:
I suggest `siteSearch`, assuming that search results may aggregate catalog, CMS and syndicated content.

Reply:
> I suggest siteSearch, assuming that search results may aggregate catalog, CMS and syndicated content.

I think that's getting closer, but 2 concerns:

1. I don't think either of these queries are designed to be a generic entity search (they both return data about products). Did I read the types wrong, or is the suggestion that we expand the design of this along with the name?
2. `siteSearch` could lead folks to believe, based on the name, that it lets you search across _all_ different ["sites"](https://devdocs.magento.com/guides/v2.3/config-guide/multi-site/ms_over.html) within

Reply:
We will add categories there later

Reply:
I think my preference would be to avoid taking `search` right now, since this search is only for products. I think we should leave names like `search` for a time when we might have some global search feature (one search for catalogs/products/cms/etc)

Reply:
This was the exact purpose of the query. I removed it for now as we don't plan to use it in the nearest future. We can always add it.